change/@fluentui-web-components-037cbaec-4ee8-4f24-8cb7-1ae8ea8546f4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=32852&commitId=fca69d08245b6b8878b56bbd3b1810533151f9c4&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|284| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>